dev/PushNotifications/PushNotificationsLongRunningTask/externs.h

```diff
@@ -0,0 +1,18 @@
+﻿#pragma once
+#include "pch.h"
+#include "platform.h"
+
+wil::unique_event& GetWinMainEvent();
+
+WpnLrpPlatformImpl* GetPlatform();
+
+HRESULT InitializePlatform();
```

Comment:
Why externs?

Reply:
I read through the code. In reality we don't need any of these. The timer APIs can be part of platform. The rest can be part of the Factor instance that we register with COM. And the factory instance can essentially wrap the Platform object in question.